vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/dataview/ComboBoxListDataView.java

```diff
@@ -46,10 +48,14 @@
      *            the data communicator of the ComboBox, not <code>null</code>
      * @param comboBox
      *            the ComboBox component, not <code>null</code>
+     * @param dataChangedCallback
+     *            callback, which is being invoked when the ComboBox's filtering
+     *            or sorting changes, not <code>null</code>
      */
     public ComboBoxListDataView(DataCommunicator<T> dataCommunicator,
-            ComboBox<T> comboBox) {
-        super(dataCommunicator::getDataProvider, comboBox);
+            ComboBox<T> comboBox,
+            SerializableBiConsumer<SerializablePredicate<T>, SerializableComparator<T>> dataChangedCallback) {
```

Comment:
I don't know why I didn't pay attention to this before, but _data changed callback_  seems a bit odd name when it is about sorting or filtering changing

Reply:
Renamed here and on the flow side also.